src/tls13.c

```diff
@@ -7784,6 +7785,55 @@ static WC_INLINE void EncodeSigAlg(byte hashAlgo, byte hsType, byte* output)
     }
 }
 
+#ifdef WOLFSSL_DUAL_ALG_CERTS
+/* These match up with what the OQS team has defined. */
+#define HYBRID_SA_MAJOR 0xFE
+#define HYBRID_P256_DILITHIUM_LEVEL2_SA_MINOR    0xA1
+#define HYBRID_RSA3072_DILITHIUM_LEVEL2_SA_MINOR 0xA2
+#define HYBRID_P384_DILITHIUM_LEVEL3_SA_MINOR    0xA4
+#define HYBRID_P521_DILITHIUM_LEVEL5_SA_MINOR    0xA6
+#define HYBRID_P256_FALCON_LEVEL1_SA_MINOR       0x0C
+#define HYBRID_RSA3072_FALCON_LEVEL1_SA_MINOR    0x0D
+#define HYBRID_P521_FALCON_LEVEL5_SA_MINOR       0x0F
+
+static WC_INLINE void EncodeDualSigAlg(byte sigAlg, byte altSigAlg,
```

Comment:
I would remove `WC_INLINE`. Let compiler decide.

Reply:
Sounds good to me. Fixed.